chcemvediet/apps/inforequests/templates/inforequests/detail/action/attachments.html

```diff
@@ -13,10 +13,25 @@
 
 <div class="pln-attachments-list">
   {% for attachment in action.attachments %}
-    <div class="pln-attachment">
-      <a href="{% url 'inforequests:download_attachment' attachment=attachment %}">
-        {{ attachment.name }}
-      </a>
-    </div>
+      {% if request.user == inforequest.applicant %}
+        <div class="pln-attachment">
+          <a href="{% url 'inforequests:download_attachment' attachment=attachment %}">
+            {{ attachment.name }}
+          </a>
+        </div>
+      {% elif attachment.attachmentfinalization_set.first %}
+        {% for attachment_finalization in attachment.attachmentfinalization_set.all %}
```

Comment:
Co tak si pozriet dokumentaciu k Djangu a zamysliet sa? https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#for-empty

```
{% for athlete in athlete_list %}
    <li>{{ athlete.name }}</li>
{% empty %}
    <li>Sorry, no athletes in this list.</li>
{% endfor %}
```

Usetris zbytocny druhy select do DB.

Reply:
:ok: